__tests__/integration/one-app.spec.js

```diff
@@ -1578,9 +1578,6 @@ describe('Tests that require Docker setup', () => {
         date: [
           expect.any(String),
         ],
-        'expect-ct': [
```

Comment:
expect-ct is deprecated - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT